### PR TITLE
(2196) Feature: include adjustments in calculation of variance

### PR DESCRIPTION
**Pull request by mec**
## Changes in this PR
Include adjustments of type `Actual` in the variance calclulation.

Also moves some content to the locales.

https://trello.com/c/FX0NPZux

### CODE REVIEW

**Comment by edavey** [comment]
[Include adjustments in calculation of variance (Activity#variance_for_report_financial_quarter)](https://trello.com/c/FX0NPZux/2196-include-adjustments-in-calculation-of-variance-activityvarianceforreportfinancialquarter)

File: spec/models/activity_spec.rb [hunk 1]
```diff
@@ -981,6 +981,25 @@
...
+
+    it "does NOT include Adjustments of type 'Actual' for a different financial quarter in the calculation" do
+      project = create(:project_activity, :with_report)
+      report = Report.for_activity(project).first
+      create(:actual, parent_activity: project, value: 130.20, report: report, **current_quarter)
+      create(:adjustment, :actual, parent_activity: project, value: -30, report: report, **current_quarter.pred)
+
+      expect(project.actual_total_for_report_financial_quarter(report: report)).to eq(130.20)
+    end
+
```
Comment (edavey) [inline spec/models/activity_spec.rb:1002]: super-clear description of this behaviour, thks!

As an aside, I note that the somewhat magical way that a report can be initialised with an optional financial quarter is not really described in the unit tests. I think I will have a go at that as a "documentation" task!

File: config/locales/models/report.en.yml [hunk 2]
```diff
@@ -33,15 +33,25 @@ en:
...
+        heading: Summary
+      activities:
+        heading: Activities
+      forecasts:
+        heading: Forecasts
+      actuals:
+        heading: Actuals
+      variance:
+        heading: Variance
+        body: This tab shows the variance between forecasted and actual spend, including any adjustments, in this reporting cycle.
```
Comment (edavey) [inline config/locales/models/report.en.yml:46]: I suspect that the change in behaviour contained here would be much clearer if you broke this down into 2 commits:

1. refactor with no changes to content
2. change in content to add the variance body "This tab shows the..."

**Review by edavey — APPROVED** [review]
Nice!

Just one suggestion to maybe make the refactor element separate from the content-change element....

File: spec/models/activity_spec.rb (hunk 1 above)
Comment (mec) [reply spec/models/activity_spec.rb:1002]: Is it not covered in the `HasFinancialQuarter` spec?

**Pull request merged**